src/utils/api/auth.ts

```diff
@@ -1,8 +1,10 @@
 import { authController } from "../libs/requestUrls";
 import RequestApiV2 from "../libs/requestApi";
+import { AxiosResponse } from "axios";
+import { authRes } from "types/api";
 
 class Auth {
-  async login(password: string) {
+  async login(password: string): Promise<void | AxiosResponse<authRes>> {
```

Comment:
```suggestion
  async loginByPassword(password: string): Promise<void | AxiosResponse<authRes>> {
```

무엇으로 로그인하는지 명시적으로 적어주는 것이 가독성에 더욱 도움이 되지않을까 싶습니다.
로그인이 한개가 아니기때문인 이유도 있습니다.